### PR TITLE
More explicit 3.4, 3.5, 3.6 & 3.7 support

### PR DESCRIPTION
**Pull request by jheld**

### CODE REVIEW

**Comment by jheld** [comment]
@km-pg can you check against this branch?

**Comment by km-pg** [comment]
This worked for me without error with Python 3.7 and Django 1.11.18

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ env:
     - DJANGO="django>=1.11,<2"
     - DJANGO="django>=2.0,<2.1"
     - DJANGO="django>=2.1,<2.2"
+    - DJANGO="django>=1.11.17,<2.2"
```
Comment (km-pg) [inline .travis.yml]: Not sure what you are testing with this line. The latest 2.1? That's already covered in the previous line.

Comment (jheld) [reply .travis.yml]: I feel like I need it for py3.7, which is 1.11.17 and higher.

**Pull request closed without merging**